src/components/AutoCompleteInput/AutoCompleteInput.tsx

```diff
@@ -59,29 +68,32 @@ type Props = {
   /**
    * Ref callback to set reference on input box
    */
-  setInputBoxRef: React.RefObject<TextInput>;
+  setInputBoxRef: React.RefObject<TextInputType>;
```

Comment:
Why cant this just be TextInput? I remember you said a styled component thing, but I am not seeing an error.

Reply:
It was throwing me an error and I think it was cause it's referencing the class TextInput instead of the type